python/spec/dependabot/python/metadata_finder_spec.rb

```diff
@@ -317,6 +317,12 @@
 
       it { is_expected.to eq("https://github.com/celery/celery") }
     end
+
+    context "when the dependency source is in project_urls" do
+      let(:pypi_response) { fixture("pypi", "pypi_response_project_urls_source.json") }
+
+      it { is_expected.to eq("https://github.com/bedford/django-split-settings") }
```

Comment:
this seems to be a private link too?

Reply:
I used bedford as a placeholder but I've changed it to xxxxx to avoid confusion.